internal/clients/stubs/ec2_stub.go

```diff
@@ -46,29 +46,31 @@ func (mock *EC2ClientStub) Status(ctx context.Context) error {
 }
 
 func (mock *EC2ClientStub) ImportPubkey(ctx context.Context, key *models.Pubkey, tag string) (string, error) {
-	ec2KeyID := fmt.Sprintf("key-%d", len(mock.Imported))
+	ec2KeyID := fmt.Sprintf("key-%d", len(mock.Imported)+1)
 	keyName := key.Name // copy the name
+	var tags []types.Tag
+	if tag != "" {
+		tags = append(tags, types.Tag{Key: ptr.To("rhhc:id"), Value: &tag})
```

Comment:
I remember lzap explained it is better to initialize an array to a certain length and maybe add capacity. 
is it possible here?